pipelines/pipeline_dada2.py

```diff
@@ -231,7 +230,7 @@ def filterAndTrim(infile, outfile):
         paired = ""
         infiles = [infile]
 
-    tmpdir = P.get_temp_dir()
+    tmpdr = P.get_temp_dir()
```

Comment:
I can't work out why this has changed from tmpdir to tmpdr?

Reply:
that was an accident. I've changed it back on my local version already. I've also improved the yml2Table function. I'll push these two changes out today.

Reply:
Great. If you push these then I will merge into master